Gruntfile.js

```diff
@@ -279,6 +281,14 @@ module.exports = function(grunt) {
         ],
         dest: 'webapp/node_modules_backup',
       },
+      'cht-libraries-to-patch': {
```

Comment:
Or be a step in "rendering" login page.

Reply:
It's a bit tricky to load this script differently because the login page has no build step like webapp or admin, it's basic html/css/js and we can't directly bundle node_modules with an `import`/`require()`

So I followed your advice and made grunt copy the `lib-bowser.js` file at startup, no need for the symlink and the grunt-contrib-copy patch. Less code \o/